src/heev.cc

```diff
@@ -111,14 +113,14 @@ void heev(
     Lambda.resize(n);
     std::vector<real_t> E(n - 1);
     Matrix<scalar_t> V;
+    // Matrix to store Householder vectors.
+    // Could pack into a lower triangular matrix, but we store each
+    // parallelogram in a 2nb-by-nb tile, with nt(nt + 1)/2 tiles.
+    int64_t vm = 2*nb;
+    int64_t nt = A.nt();
+    int64_t vn = nt*(nt + 1)/2*nb;
+    V = Matrix<scalar_t>(vm, vn, vm, nb, 1, 1, A.mpiComm());
```

Comment:
This is okay for the moment, but this uses O(n^2) memory on _every_ rank, right? So we eventually need a more scalable solution.